operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -1515,6 +1527,34 @@ func (rc *ReconciliationContext) findStartedNotReadyNodes() (bool, error) {
 	return false, nil
 }
 
+func (rc *ReconciliationContext) copyPodCredentials(pod *corev1.Pod, jksBlob []byte) error {
```

Comment:
no pod stuff ended up getting used?

Reply:
I think this belongs in the/an init container at this point. This gets us encryption without the fancy peer verification, it's true, but I would like to get this in so it can be iterated on.

Reply:
👍